src/System.Net.Primitives/tests/FunctionalTests/CookieTest.cs

```diff
@@ -337,5 +337,26 @@ public static void ToString_Compare_Success()
             c.Port = "";
             Assert.Equal("name=value; $Path=path; $Domain=domain; $Port", c.ToString());
         }
+        
+        [Theory]
+        [MemberData(nameof(HashCodesTestData))]
+        public static void HashCodes_ShouldWorkLikeTuple(string name, string value, string path, string domain, int version)
```

Comment:
@CIPop Unfortunately, it's not possible to hard-code these values as .NET Core uses randomized string hashing: [dotnet/coreclr#229](https://github.com/dotnet/coreclr/pull/229#issuecomment-216311479) meaning that `"string".GetHashCode()` will be different every time you run the app.

Would it be ok to assert that we use the same algorithm as the `Hash` helper class I'm going to introduce, e.g.

```cs
Assert.Equal(Hash.CombineHashCodes(name, value, path, domain, version), cookie.GetHashCode());
```

or just leave out the test altogether? (As @ellismg points out, if we change the algorithm in the future these tests will fail.)